### PR TITLE
Implement BaseURL setting

### PR DESCRIPTION
**Pull request by geNAZt**
Move from github.com imports to relative ones for easier forking

### CODE REVIEW

File: app/webhook/webhook.go [hunk 1]
```diff
@@ -70,22 +72,22 @@ func (w *Webhook) Post(entry ReportListEntry) {
...
 
 	messageText := make([]string, 0, 20)
 	for _, entry := range w.reportList {
-		messageText = append(messageText, fmt.Sprintf("<https://crash.pmmp.io/view/%d|#%d: %s>", entry.ReportId, entry.ReportId, entry.Message))
+		messageText = append(messageText, fmt.Sprintf("<%s/view/%d|#%d: %s>", w.baseURL, entry.ReportId, entry.ReportId, entry.Message))
 	}
 	t := strings.Join(messageText, "\n")
 	if w.reportCount > uint32(cap(w.reportList)) {
-		t += fmt.Sprintf("\n\n%d more reports not shown. <%s|View the full list>", w.reportCount - uint32(cap(w.reportList)), listUrl)
+		t += fmt.Sprintf("\n\n%d more reports not shown. <%s|View the full list>", w.reportCount-uint32(cap(w.reportList)), listUrl)
```
Comment (SOF3) [inline app/webhook/webhook.go]: Is there any specification on the spacing style here?

Comment (dktapps) [reply app/webhook/webhook.go]: Personally I've been using spaces for as long as I can remember. The cramped style like this hurts readability in my opinion.

Comment (SOF3) [reply app/webhook/webhook.go]: GoLand removes spaces automatically sometimes, but I don't understand when exactly.

Comment (Sandertv) [reply app/webhook/webhook.go]: `go fmt` removes spaces here iirc

Comment (dktapps) [reply app/webhook/webhook.go]: Oh boy, opinionated default styling. Is there a way to change that?

Comment (SOF3) [reply app/webhook/webhook.go]: @dktapps One of the philosophies is "Stop arguing over which style to use; use our default style guide!" gofmt is even more authoritarian than me!

**Comment by dktapps** [comment]
This PR is long since stale, and the change it claimed to implement has been long since done (dd67a80cdfe597a999c9d65c11a6147daaa8ef4a).

**Pull request closed without merging**